### DISCUSSION TITLE
sync tab auto switch from on to off when starting browser

### DISCUSSION THREAD

**Discussion by candrapersada** [floccusaddon/floccus#1081] [General]
is there an option to auto disable tab sync when starting browser,
so that tabs don't disappear from local & cloud when you start opening browsers?
![image](https://user-images.githubusercontent.com/36833664/159113170-acbd15c4-4b44-480e-8159-05084841dc49.png)

**Comment by marcelklehr** [comment]
Do tabs disappear for you upon opening the browser when Auto-Sync is off?

**Comment by candrapersada** [comment]
what I mean is the tab that I save in the cloud disappears when opening the browser when Auto-Sync is on
and is there an auto switch option from on to off when starting browser
![image](https://user-images.githubusercontent.com/36833664/159119174-34987ddf-52a1-40e2-92a3-823b7ce75a76.png)

**Reply by marcelklehr** [reply]
I recommend not using auto-sync together with tab sync at the moment. Automatically switching auto-sync off on browser start would fix this, but is really unexpected. Perhaps the best way to fix the experience with tab sync is adding an option to only sync an account on browser start. Then you can have 

1. An account that always pushes tabs
2. An account that only pulls tabs on browser start

**Comment by candrapersada** [comment]
> 1. An account that always pushes tabs
> 2. An account that only pulls tabs on browser start

how to only pulls tabs on browser start and always pushes tabs?

**Reply by marcelklehr** [reply]
That was a hypothetical, it's not yet possible
